server/src/project/test/document.test.ts

```diff
@@ -82,6 +82,27 @@ describe('ModelicaDocument', () => {
     assert.equal(document.getText().trim(), UPDATED_TEST_PACKAGE_CONTENT.trim());
   });
 
+  it('can update incrementally', () => {
+    const textDocument = createTextDocument('.', TEST_PACKAGE_CONTENT);
+    const tree = project.parser.parse(TEST_PACKAGE_CONTENT);
+    const document = new ModelicaDocument(project, library, textDocument, tree);
+    document.update(
+      '1.0.1',
+      {
+        start: {
+          line: 1,
+          character: 22,
+        },
+        end: {
+          line: 1,
+          character: 27,
+        },
+      }
+    );
+
+    assert.equal(document.getText().trim(), UPDATED_TEST_PACKAGE_CONTENT.trim());

```

Comment:
Add an additional assert to test that the tree indeed is updated. In this case it's hard to tell, so maybe do an update that does change the tree. Maybe adding another annotation or model or so.

Otherwise we aren't sure that the tree was updated.

Reply:
Ups, wrong button. I wanted to request changes on my review^^. Please check if you can update the test.

Reply:
Like this?